### PR TITLE
RK-2688 - Better readme

### PR DESCRIPTION
**Pull request by amits1995**

### CODE REVIEW

**Comment by amits1995** [comment]
@LiranLast WDYT?

File: README.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+The API is protected by a self generated token.
 
-There are 3 components to this electron app:  
+# Security
+Explorook uses a self-generated, crypto-safe, 128-bit token which guarantess that any third party trying to access Explorook's API is blocked  
+Explorook only allows read-only access and only to folders the user specifies (and their subfolders)  
+Folders traversal are not allowed  
+Explorook does not send any information about the user's source code to any server
+
+Explorook spawns three processes (one main and two renderers):
```
Comment (LiranLast) [inline README.md]: + Rookout is only accessible from localhost

Comment (amits1995) [reply README.md]: K

**Pull request merged**